Add BN and QAT Restructrure news post (#2162)
ppy/osu-wiki: news/2019-04-10-bn-and-qat-restructure.md

+ Beatmaps that have been vetoed will be mediated by a random jury made up of members of the BN. The NAT will oversee these mediations. 

The [Beatmap Nominator rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules) have been updated accordingly for their new tasks. Moreover, work on a score system as a guideline to evaluate Beatmap Nominators starts now and you can expect a new proposal detailing our plans for that soon!

Additionally to this, the next time BN Applications open, they will stay open indefinitely. More information about how to apply will be released soon.

As osu! is a community driven game, these changes were made to give members of the community more control over what reaches the ranked section. We expect to see improvements in the way beatmap quality is managed, particularly with controversial maps, where the outcome will be more in line with the desires of the community.

> With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909) [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), and [Protastic101](https://osu.ppy.sh/users/6712747) who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.

Reviewer: ```suggestion
With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), and [Protastic101](https://osu.ppy.sh/users/6712747) who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.
```
missing a comma somewhere

---
layout: post
title: "BN and QAT Restructure"
date: 2019-04-10 23:30:00 +0000
---

Changes to the ranking process are nigh! With recent discussions about [changes to the Quality Assurance Team and Beatmap Nominators](https://osu.ppy.sh/community/forums/topics/862002), we are proud to present to you the following changes.

![](/wiki/shared/news/banners/beatmap-management.jpg)

+ The QAT is gone and has been replaced by the new [Nominator Administration Team](https://osu.ppy.sh/groups/7).
+ The Beatmap Nominators (BN) are responsible for disqualifying beatmaps and mediating vetoes from now on, in addition to their task of pushing beatmaps towards the ranked category.
+ The Nominator Administration Team (NAT) are the moderators of the Beatmap Nominators. They are responsible for BN Applications, creating tests for the applicants and for evaluating the current members. They will still have their moderation privileges although it is not their primary focus. 
+ Beatmaps that have been vetoed will be mediated by a random jury made up of members of the BN. The NAT will oversee these mediations. 

> + Probationary Beatmap Nominators now have their own usergroup. These members will have less features compared to the rest of the BNs.

Reviewer 2: maybe just phrase as "Probationary Beatmap Nominators won't be able to \<do x things\>", the usergroup won't be public so it's kinda useless information

probably works either way though.

Author: I added that information in the next paragraph. Was told that even if probation has been for a while, since it's an actual osu-web change it should be mentioned in the news post, and to give as much info about it as we can.

Reviewer 2: then it's also backwards, the new usergroup is for non-probationary bns (could still change, waiting for review about that)

Author: ```suggestion
+ The current Beatmap Nominator usergroup will be used for probationary Beatmap Nominators. A new usergroup will be made for regular members of the Beatmap Nominators which will grant access to the disqualification button.
```

Reviewer 2: I don't think it's a good idea to explain implementation details in a news post, if people are curious they can easily find details in osu-web or just by poking around on the website

anyway if this stays you'll have to update it again https://github.com/ppy/osu-web/pull/4402

Author: Should already be updated (different to what I suggested above). Got criticism from higher up that the news post lacked detail, so here we are.

Reviewer 2: yea nvm I think i was looking at an intermediate version it is correct now

ig it makes sense to include this info now that both groups are intended to be public, but next time ask them to comment here so we are not changing things without a (public) reason

As osu! is a community driven game, these changes were made to give members of the community more control over what reaches the ranked section. We expect to see improvements in the way beatmap quality is managed, particularly with controversial maps, where the outcome will be more in line with the desires of the community.

With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), and [Protastic101](https://osu.ppy.sh/users/6712747) who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.

On a more uplifting note, please welcome [Ascendance](https://osu.ppy.sh/users/2931883) to the NAT, who will be focusing their efforts on the affairs of the osu!catch nominators!

> 

Reviewer 3: ```suggestion
For more information on what Beatmap Nominators do and how to join them, have a look at their [wiki page](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators)!

```

As osu! is a community driven game, these changes were made to give members of the community more control over what reaches the ranked section. We expect to see improvements in the way beatmap quality is managed, particularly with controversial maps, where the outcome will be more in line with the desires of the community.

With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), and [Protastic101](https://osu.ppy.sh/users/6712747) who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.

On a more uplifting note, please welcome [Ascendance](https://osu.ppy.sh/users/2931883) to the NAT, who will be focusing their efforts on the affairs of the osu!catch nominators!

> For more information on what Beatmap Nominators do and how to join them, have a look at their [wiki page](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators)!

Reviewer 2: needs another newline if you want a new paragraph here

+ The current Beatmap Nominator usergroup will be used for probationary Beatmap Nominators. A new usergroup will be made for regular members of the Beatmap Nominators which will grant access to the disqualify button.

The primary focus of probationary BNs will be to nominate beatmaps, and so will not have the ability to disqualify beatmaps or take part in mediating vetoes. Moreover, they will be unable to nominate beatmaps with more than one game mode, and a beatmap cannot be qualified by two probationary BNs. New members of the BNs will be placed in probation for approximately one month before their performance is reviewed by the NAT. They will then be promoted to a regular BN, remain in probation, or be removed from the group completely. Members of the regular BNs may also be demoted to probation if their work is found to not meet standards.

The [Beatmap Nominator rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules) have been updated accordingly for their new tasks. Moreover, work on a score system as a guideline to evaluate Beatmap Nominators starts now and you can expect a new proposal detailing our plans for that soon!

> Additionally to this, BN applications will be open indefinitely from the time of this news post! We have a [new website](http://bn.mappersguild.com/) to deal with BN affairs including applications and reporting BNs. Special thanks to [pishifat](https://osu.ppy.sh/users/3178418) and [Milan-](https://osu.ppy.sh/users/1052994) for developing the website. 

Reviewer 4: mapping-related reports in general, not just reporting BNs (stolen maps, inappropriate content, and bn/nat conduct)

also last sentence credits dont need to exist

> BN applications will still work roughly the same as last time, where you are required to pass an activity check and to complete a test on the RC to complete your application. Additionally, there will be a 90 day cooldown for submitting applications once you submit. The results of your application should be sent out as soon as they are processed, however, we expect a larger influx of applications during the initial weeks. Check out [the website](http://bn.mappersguild.com/) to submit an application to join the BN!

Reviewer 4: RC --> Ranking Criteria

With these changes, we unfortunately have had to say farewell to [Fycho](https://osu.ppy.sh/users/1876867), [Gabe](https://osu.ppy.sh/users/654108), [Hobbes2](https://osu.ppy.sh/users/8157492), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [p3n](https://osu.ppy.sh/users/123703), [Regraz](https://osu.ppy.sh/users/3076909), [Raiden](https://osu.ppy.sh/users/2239480), [JBHyperion](https://osu.ppy.sh/users/4879508), [Protastic101](https://osu.ppy.sh/users/6712747), [-Kazu-](https://osu.ppy.sh/users/920861), and [Noffy](https://osu.ppy.sh/users/1541323), who will not be joining the NAT. Nonetheless, we greatly appreciate their time in the QAT and thank them all for their hard work up until now.

On a more uplifting note, please welcome [Ascendance](https://osu.ppy.sh/users/2931883) to the NAT, who will be focusing their efforts on the affairs of the osu!catch nominators!

> For more information on what Beatmap Nominators do and how to join them, have a look at their [wiki page](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators)!

Reviewer 4: isnt this covered differently in a paragraph above? maybe just incorporate the wiki link there

Author: Was asked to have this explicitly and in a clear place so.
